lblod/app-digitaal-loket
# Add authentication cleanup job

## Ticket Number

DL-6077

## PR Description

Adds a cleanup job to remove authentication data from **failed**/**finished** automatic submissions jobs and from download url jobs which are in a **final** state.

## How to test

Run the following command to manually execute the cleanup job:
```bash
docker compose exec dbcleanup curl -X GET "http://localhost/runCronjob?cronJobID=4d3b2cec-68a7-4703-9cbc-0a8ca6c29539"
```
and check the `dbcleanup` logs. The service should print out the query to execute and whether it found no resources or it was executed.
